views/templates/admin/smaily_configure.tpl

```diff
@@ -181,6 +181,35 @@
               </p>
             </div>
           </div>
+          <div class="form-group">
+            <label class="control-label col-lg-2">
+              {l s="Trigger opt-in on customer sign-up" mod='smailyforprestashop'}
+            </label>
+            <div class="col-lg-10">
+              <span class="switch prestashop-switch fixed-width-lg">
+                <input type="radio" name="SMAILY_OPTIN_ENABLED" id="Enable" value="1" {($smaily_optin_enabled == 1) ? 'checked' :'' }>
+                <label for="Enable">{l s="Yes" mod='smailyforprestashop'}</label>
+                <input type="radio" name="SMAILY_OPTIN_ENABLED" id="Disable" value="0" {($smaily_optin_enabled == 0) ? 'checked' :'' }>
+                <label for="Disable">{l s="No" mod='smailyforprestashop'}</label>
+                <a class="slide-button btn"></a>
+              </span>
+              <p class="help-block">
+                {l s="Opt-in will only be triggered when customer creates an account & signs-up for newsletter. Changes to newsletter subscription in admin won't trigger an opt-in." mod='smailyforprestashop'}
+              </p>
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="control-label col-lg-2">
+              {l s="Automation to trigger on customer sign-up" mod='smailyforprestashop'}
+            </label>
+            <div class="col-lg-10">
+              <select name="SMAILY_OPTIN_AUTORESPONDER" id="SMAILY_OPTIN_AUTORESPONDER" data-selected-id={$smaily_optin_autoresponder}>
+                <option value="">
+                  {l s="No Autoresponder" mod='smailyforprestashop'}
```

Comment:
Perhaps it would be better to use a more explanatory phrase here. So instead of "No Autoresponder" it should say "Trigger any opt-in automation".

Reply:
Maybe it would be even better to use the same solution like in abandoned cart autoresponder?
```
<option value="">
    {l s="Select Autoresponder" mod='smailyforprestashop'}
</option>
```

Reply:
Yes, if we would have Abandoned Cart trigger, but unfortunately we do not. Abandoned Cart shouldn't trigger opt-in automations. Maybe we should even make Abandoned Cart autoresponder selection mandatory?

Reply:
I mean we're using autoresponder instead of automation on every plugin, only using automation in documentation. I am suggesting we keep with the consistency.

Reply:
> Maybe we should even make Abandoned Cart autoresponder selection mandatory?
I think we should make it  mandatory

Reply:
Oh, OK. Yeah, we should use the same terms everywhere. Change the term to "Trigger any opt-in autoresponder" then (omitting "opt-in" might make user think that any autoresponder is triggered, no matter what the trigger is). Most likely it will change in the future to "automation", but at least it is consistent for now.